test_config.placeholder.json

```diff
@@ -0,0 +1,14 @@
+{
+  "endpoints": [
```

Comment:
Did you choose `snake-case` for any particular reason? Most of our existing samples that use JSON use `PascalCase`.

Reply:
I chose snake-case as it was what I had used for the OMF Python sample. I am not opposed to changing it to PascalCase since that is what seems to be the standard for our samples. I will need to circle back to the OMF sample at some point to change it there as well.